hasura/graphql-engine
# Standard capitalization of documentation headings (close #8885)

### Description
This commit changes all of the markdown headings in the documentation to be capitalized as per the styling guidelines [here](https://hasura.io/docs/wiki/style/headings/) in relation to issue #8885.

### Changelog



__Component__ : documentation

__Type__: enhancement

__Product__: community-edition

### Related Issues

#8885 
#9072 

### Solution and Design

To achieve this task and to avoid both typos and/or missed files, I threw a quick program together to do this automatically which can be found [here](https://github.com/IllogicalTree/markdown-style-formatter) for those curious. 

### Steps to test and verify

A large selection of the changed documentation files have been manually reviewed to verify the change works as intended.

#### Catalog upgrade

Does this PR change Hasura Catalog version?

Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
